src/config.h.in

```diff
@@ -0,0 +1,4 @@
+#if XAPIAN_MAJOR_VERSION >= 2 || XAPIAN_MINOR_VERSION >=3 || (XAPIAN_MINOR_VERSION == 3 && XAPIAN_REVISION >= 6)
```

Comment:
Almost. You also need to delete `src/config.h.in` as it's no longer used, drop it as `AC_CONFIG_HEADER()` in `configure.ac`, and I don't think `XAPIAN_OP_MAX_SUPPORT` is defined anywhere at the moment?